try {
    isFinite.apply.arguments;
} catch (error) {
    throw new EvalError("Se ha prodicido un error linea 100")
}